.gitignore

```diff
@@ -12,6 +12,8 @@ build/*
 # Cairo dependencies
 cairo-lang
 venv
+snos-env
+.python-version
```

Comment:
Why is this being ignored?

Reply:
Because that is stuff generated by the setup

Reply:
I was referring to `.python-version` - it fixes the python version to use. Is there not a specific version of python that must be used? Your setup-cairo script is mandating verion 3.9.15.